cmd/zfs/zfs_main.c

```diff
@@ -8059,6 +8065,18 @@ zfs_do_project(int argc, char **argv)
 	return (ret);
 }
 
+/*
+ * Display version message
+ */
+static int
+zfs_do_version(int argc, char **argv)
+{
+	if (zfs_version_print() == -1)
+		exit(errno);
```

Comment:
`zfs(8)` should exit with `1` on error, let's `return (1);`.

```
EXIT STATUS
     The zfs utility exits 0 on success, 1 if an error occurs, and 2 if invalid
     command line options were specified.
```

Reply:
will do

Reply:
done